docs/introduction-to-smart-contracts.rst

```diff
@@ -94,7 +94,7 @@ registering with username and password - all you need is an Ethereum keypair.
 
         // This is the constructor whose code is
         // run only when the contract is created.
-        function Coin() public {
```

Comment:
This will be fixed in the PR that enforces this syntax.  Could you please remove it from your PR as it is also unrelated to the rest?

Reply:
Sure.